tests/molecules.py

```diff
@@ -64,6 +64,8 @@ def loadall(fname: str) -> Tuple[List[Any], List[molecule.Molecule]]:
 obsdf = [obbenzene, obpyridine, obethanol, obdialanine]
 sdf = [benzene, pyridine, ethanol, dialanine]
 
+obmuparfostat, muparfostat = load("muparfostat.sdf")
```

Comment:
This file has been remove in #118. `muparfostat` will need to become a fixture in `confutes.py` (same as `benzene` or `pyridine`).